packages/icon/components/Icon.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from "react";
+import { cx } from "emotion";
+import { tintSVG } from "../../shared/styles/styleUtils";
+import {
+  textColorPrimary,
+  iconSizeXs,
+  iconSizeS,
+  iconSizeM,
+  iconSizeL,
+  iconSizeXl,
+  iconSizeXxl
+} from "../../design-tokens/build/js/designTokens";
+import { SystemIcons } from "../../icons/dist/system-icons-enum";
+import { ProductIcons } from "../../icons/dist/product-icons-enum";
+import { icon } from "../style";
+
+export enum IconSizes {
+  XS = iconSizeXs as any,
+  S = iconSizeS as any,
+  M = iconSizeM as any,
+  L = iconSizeL as any,
+  XL = iconSizeXl as any,
+  XXL = iconSizeXxl as any
```

Comment:
Is having `as any` here really necessary?

Reply:
I was having trouble with this one. String enums that use constants as an initializer are a big pain